src/test/lut.test.js

```diff
@@ -1,8 +1,12 @@
 import { expect } from "chai";
 
-import Histogram from "../Histogram.js";
+import Histogram from "../Histogram.ts";
 import VolumeMaker from "../VolumeMaker.js";
 
+function clamp(val, cmin, cmax) {
+  return Math.min(Math.max(cmin, val), cmax);
+}
+
```

Comment:
Maybe throw this in a util file if it's going to be used in multiple places (here and in Histogram.ts)?

Reply:
The only reason I didn't is that the repeated use is in a unit test.  One more repetition and I will refactor it :)